test/testlibrary.cpp

```diff
@@ -827,8 +827,8 @@ class TestLibrary : public TestFixture {
         Library library;
         ASSERT(loadxmldata(library, xmldata, sizeof(xmldata)));
 
-        Library::Container& A = library.containers["A"];
-        Library::Container& B = library.containers["B"];
+        const Library::Container& A = library.containers["A"];
```

Comment:
I think this false negative is already tracked in https://trac.cppcheck.net/ticket/12211.